src/preamble_minimal.js

```diff
@@ -146,6 +146,9 @@ var runtimeInitialized = false;
 
 #include "runtime_math.js"
 #include "memoryprofiler.js"
+#if EXCEPTION_STACK_TRACES && !WASM_EXCEPTIONS
```

Comment:
I think it might be more clear to put the `#if EXCEPTION_STACK_TRACES && !WASM_EXCEPTIONS` inside the `src/runtime_exceptions.js` file, so it obvious the reader of that file that code is there to support stack traces, but only for emscripten EH.

Putting the check around the include itself seems like separating that information from where it would naturally live.

Reply:
Ah, I think I misunderstood your previous review comment: https://github.com/emscripten-core/emscripten/pull/18535#discussion_r1106288666. Let me restore this to its previous state.

Reply:
04d643b1e7ba3ad6022933dd356477d98042e800